app/assets/stylesheets/settings/_print.scss

```diff
@@ -1,11 +1,17 @@
 @media print {
   @page {
-    size: A4 portrait;
+    size: A4;
+    // margin: 0;
   }
 
-  body {
-    width: 300mm !important;
-    height: 99%;
+  html, body {
+    width: 300mm;
```

Comment:
umm. Be careful. A4 width is 210 in portrait. So setting body to 300mm, you can have problems.